spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -212,3 +211,12 @@ n_samples_per_recording_entry = 100
 # True  Which will make sure the board is off at this point
 # False Which will make sure the board is on at this point
 turn_off_board_after_discovery = False
+
+[Java]
+use_java = False
+# call to start a java process.
+# If there is no jdk../bin in your class path this must include the full path
+java_call = java
+# Absolute path to where the JavaSpiNNaker git repository is located.
+# If none it will be assumed to be in the same parent directory as SpiNNFrontEndCommon
+javaspinnaker_path = None
```

Comment:
```suggestion
java_spinnaker_path = None
```

Reply:
Note also that underscores and capitals are semantic sugar in our config files and the CamelCaseParser we use strips out underscores and lower cases they key when reading in config files and when retrieving values.

Reply:
i agree we've never been consistent in these. but thats another task issue. but trying to keep it consistent must be a good thing overall?